### PR TITLE
rename features disabled to dormancy

### PR DESCRIPTION
**Pull request by BusyJay**
disabled is a confusing features name, so change it to dormancy.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -334,7 +334,7 @@ fn set(
 }
 
 #[macro_export]
-#[cfg(not(feature = "disabled"))]
```
Comment (queenypingcap) [inline src/lib.rs]: How about changing the name to "enable_fail_point"? Then the first condition can be:
#[cfg(feature = "enable_fail_point")]
And the second the condition:
+#[cfg(not(feature = "enable_fail_point"))]

Comment (BusyJay) [reply src/lib.rs]: I prefer enabling fail points by default. When people include a crate as a dependency, it's supposed to work without any additional configuration. An extra (optional) features is just a convenient way to disable all fail points for people who don't want to put `#[cfg(features)]` around.

**Comment by overvenus** [comment]
> dormancy

I think it's Obscure, and confused me. :sweat_smile: 

I expect the default state should be "disable", just like gofail did. And I suggest rename disable to "enable".

 1. No addition features have to enable in release mode.
 2. We know it's built for testing, not for production.

**Comment by BusyJay** [comment]
> No addition features have to enable in release mode.

Then additional features have to be enabled in debug mode, I don't think there are much differences.

> We know it's built for testing, not for production.

I don't think this is the reason the functionality should be disabled by default. A more reasonable way is automatically enable it in debug mode but disable it when built in release mode.

`fail-rs` is more like FreeBSD's failpoints rather than gofail. And there seems no built-in support to disable failpoints at all.

I think features are optional. It's very strange that a crate needs to enable some features to provide its basic functionality.

File: src/lib.rs (hunk 1 above)
Comment (siddontang) [reply src/lib.rs]: How about "no_fail", I rarely see dormancy before.

**Comment by BusyJay** [comment]
ping

**Comment by queenypingcap** [comment]
LGTM.

**Pull request merged**